projects/kubernetes/build.sh

```diff
@@ -33,7 +33,10 @@ function compile_fuzzer {
   $CXX $CXXFLAGS $LIB_FUZZING_ENGINE "${fuzzer}.a" -lpthread -o "${OUT}/${fuzzer}"
 }
 
+compile_fuzzer "yaml" "FuzzDuration"
+compile_fuzzer "yaml" "FuzzMicroTime"
```

Comment:
Can you change these to FuzzDurationStrict and FuzzMicroTimeStrict?

Reply:
Done